Assets/Scripts/SS3D/Core/ApplicationInitializerSystem.cs

```diff
@@ -35,18 +36,22 @@ private void InitializeLauncher()
                 sceneToLoad = Scenes.Launcher;
             }                                  
 
+            #pragma warning disable CS4014
```

Comment:
Why ignoring this warning is okay ? Is there a reason the compiler warning is not valid ?

Reply:
That warning is there because we are not awaiting for the scene loading. It causes no issue

Reply:
Please add a comment explaining that then, so we don't have to search for the meaning of CS4014 in the future.